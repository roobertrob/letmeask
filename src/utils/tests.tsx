import { ReactElement } from 'react';
import { render as rtlRender, RenderOptions } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';

const render = (ui: ReactElement, options?: RenderOptions) =>
  rtlRender(<BrowserRouter>{ui}</BrowserRouter>, options);

export * from '@testing-library/react';
export { render };
