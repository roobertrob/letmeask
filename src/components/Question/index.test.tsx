import { render } from 'utils/tests';
import { Question } from '.';

describe('<Button />', () => {
  it('should render correctly', () => {
    const { container } = render(
      <Question content="anything" author={{ avatar: '', name: '' }} />,
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
