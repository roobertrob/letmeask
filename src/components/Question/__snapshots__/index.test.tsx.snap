// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> should render correctly 1`] = `
<div
  class="question"
>
  <p>
    anything
  </p>
  <footer>
    <div
      class="user-info"
    >
      <img
        alt=""
        src=""
      />
      <span />
    </div>
    <div />
  </footer>
</div>
`;
